examples/version.tf

```diff
@@ -0,0 +1,16 @@
+##############################################################################
+# Terraform Providers
+##############################################################################
+
+terraform {
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">= 1.41.1"
```

Comment:
examples should lock into the lowest number in the range defined in the main modules version.tf. I see that is set to `~>1.41.0` meaning you should lock into `version = "1.41.0"` in the example